03_DeviceSelectionAndSharedSources/app_resources/pplnLayoutMergeTest/shader_4.comp.hlsl

```diff
@@ -0,0 +1,14 @@
+#pragma wave shader_stage(compute)
+#include "common.hlsl"
+
+[[vk::binding(0,0)]] RWByteAddressBuffer output[];
+[[vk::binding(1,0)]] RWStructuredBuffer<SomeType> asdf;
```

Comment:
make sure you leave comments in the shaders themselves if they're supposed to be broken, like here

Reply:
this shader won't even be compatible with itself (merge with a blank pipeline intro should fail)